migrations/20231005230721_dynamic-fields.sql

```diff
@@ -0,0 +1,110 @@
+CREATE TABLE games (
+	id int PRIMARY KEY, -- Only used in db
+	name varchar(64),
+  CONSTRAINT unique_game_name UNIQUE (name)
+);
+INSERT INTO games(id, name) VALUES (1, 'minecraft-java');
+INSERT INTO games(id, name) VALUES (2, 'minecraft-bedrock');
+
+ALTER TABLE loaders ADD CONSTRAINT unique_loader_name UNIQUE (loader);
+
+ALTER TABLE mods ADD COLUMN game_id integer REFERENCES games NOT NULL DEFAULT 1; -- all past ones are minecraft-java
+ALTER TABLE loaders ADD COLUMN game_id integer REFERENCES games NOT NULL DEFAULT 1; -- all past ones are minecraft-java
+
+CREATE TABLE loader_field_enums (
+  id serial PRIMARY KEY,
+  game_id integer REFERENCES games NOT NULL DEFAULT 1,
+  enum_name varchar(64) NOT NULL,
+  ordering int NULL,
+  hidable BOOLEAN NOT NULL DEFAULT FALSE
+);
+
```

Comment:
make loader fields, loader independent, add separate table to assign version fields to loaders, they can be assigned to multiple loaders